frontend/src/routes/(authenticated)/project/[project_code]/+page.ts

```diff
@@ -107,7 +113,31 @@ export async function load(event: PageLoadEvent) {
 
   event.depends(`project:${projectCode}`);
 
+  let orgs;
+  if (userIsAdmin) {
+  const orgsResult = await client.query(graphql(`
+        query loadOrgs {
+            orgs {
+                id
+                name
+            }
+        }
+     `), {}, { fetch: event.fetch });
+    orgs = orgsResult.data?.orgs;
+  } else {
+  const myOrgsResult = await client.query(graphql(`
+        query loadMyOrgs {
+            myOrgs {
+                id
+                name
+            }
+        }
+      `), {}, { fetch: event.fetch });
+    orgs = myOrgsResult.data?.myOrgs;
+  }
+
   return {
+    myOrgs: orgs,
```

Comment:
I'd like to avoid querying the orgs list every time a project page is visited, especially since most of the time the data will never be used (it's only used when adding an org to a project). Instead pull the query code out into it's own method and call that method in `AddOrganization.openModal` before opening the form modal.